Modules/posixmodule.c

```diff
@@ -10064,6 +10069,252 @@ os_times_impl(PyObject *module)
 #endif /* HAVE_TIMES */
 
 
+#if defined(HAVE_TIMERFD_API)
+#define ONE_SECOND_IN_NS (1000 * 1000 * 1000)
+#define EXTRACT_NSEC(value)  (long)( ( (double)(value) - (time_t)(value) ) * 1e9)
+#define CONVERT_SEC_AND_NSEC_TO_DOUBLE(sec, nsec) ( (double)(sec) + (double)(nsec) * 1e-9 )
+
+static PyObject *
+build_itimerspec(const struct itimerspec* curr_value)
+{
+    double _next_expiration = CONVERT_SEC_AND_NSEC_TO_DOUBLE(curr_value->it_value.tv_sec,
```

Comment:
`next_expiration` is too long name. Why not use simple `value`?

Reply:
@serhiy-storchaka 
`value` is too common to understand the meaning.
So I chose the word by its meaning.

Reply:
@serhiy-storchaka 
Changed to value because this is just an untility.
We don't need to know the acutual meaning.